tdrs-backend/tdpservice/search_indexes/management/commands/backup_db.py

```diff
@@ -0,0 +1,53 @@
+"""Command to facilitate backup of the Postgres DB."""
+
+import os
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from tdpservice.scheduling.db_backup import main, get_system_values
```

Comment:
might be best to co-locate this management command w/ the rest of the backup script

Reply:
I like that. I made the changes!